easybuild/easyconfigs/p/PyTorch/USER.md

```diff
@@ -50,18 +61,63 @@ The EasyBuild installation with the EasyConfigs mentioned below will do three th
+
+    Those scripts don't cover all use cases for PyTorch on LUMI, but can be used as a source of
+    inspiration for your own scripts.
+
+4.  For the containers with support for virtual environments (from 20240315 on),
+    it also creates a number of commands intended to be used outside the container:
+
+    -   `start-shell`: To start a bash shell in the container. Arguments can be used
+        to, e.g., tell it to start a command. Without arguments, the conda and Python 
+        virtual environments will be initialised, but this is not the case as soon as
+        arguments are used.
+
+    -   `make-squashfs`: Make the user-software.squashfs file that would then be mounted
+        in the container after reloading the module. This will enhance performance if
+        the extra installation in user-software contains a lot of files.
+
+    -   `unmake-squashfs`: Unpack the user-software.squashfs file into the user-software
+        subdirectory of $CONTAINERROOT to enable installing additional packages.
+            
 The container uses a miniconda environment in which Python and its packages are installed.
 That environment needs to be activated in the container when running, which can be done
 with the command that is available in the container as the environment variable
 `WITH_CONDA` (which for this container it is
 `source /opt/miniconda3/bin/activate pytorch`).
 
-Example of the use of `WITH_CONDA`: Check the Python packages in the container
-in an interactive session:
+From the 20240315 version onwards, EasyBuild will already initialise the Python virtual
+environment `pytorch`. Inside the container, the virtual environment is available in
+`/user-software/venv` while outside the container the files can be found in 
+`$CONTAINERROOT/user-software/venv` (if this directory has not been removed after creating
+a SquashFS file from it for better file system performance). You can also use the 
+`/user-software` subdirectory in the container to install other software through other methods.
+
+
+## Examples
+
+Note: In the examples below you may need to replace the `standard-g` queue with a [different
+slurm partition allocatable per node]([Containers up to and including the 20240209 ones](https://docs.lumi-supercomputer.eu/runjobs/scheduled-jobs/partitions/#slurm-partitions-allocatable-by-node)) 
+if your user category has no access to `standard-g`. 
+
+
+### List the Python packages in the container
+
+#### Containers up to and including the 20240209 ones
+
+For the containers up to the 20240209 ones, this example also illustrated how the
+`WITH_CONDA` environment variable should be used.
+The example can be run in an interactive session and works even on the login nodes.
+
+In these containers, the Python packages
+can be listed using the following steps: First execute, e.g., 
 
 ```
-module load LUMI PyTorch/2.1.0-rocm-5.6.1-python-3.10-singularity-20231123
+module load LUMI PyTorch/2.2.0-rocm-5.6.1-python-3.10-singularity-20240209
```

Comment:
It is stated somewhere in the text that it doesn't matter which LUMI stack is loaded. At this point is does cause some confusion though (especially when the stack that one gets is the LUMI/22.08). Should it be stated already here (and maybe also later in the Installation section, or somewhere else) this thing that the version of the stack doesn't matter? Also it could be useful maybe to mention already here why one doesn't need to care about the partition (L/C/G/...). 
Also, I wonder why is the LUMI (LUMI/22.08) stack loaded here at all? I think somewhere it said that one can use this module also with CrayEnv, so specifically loading an old LUMI stack if not needed feels confusing.